### DISCUSSION TITLE
EXIT_ON_ZERO_ACTIVE_CONNECTIONS overrides everything?

### DISCUSSION THREAD

**Discussion by prodigy413** [istio/istio#49426] [Q&A] [answered]
Hello,

If I set EXIT_ON_ZERO_ACTIVE_CONNECTIONS to true, I think the following will happen, is this correct?
1. drainduration and terminationDrainDuration settings are overrided. (No need to think about drainduration??)
2. proxy waits until active connection becomes 0 in drain mode regardless of drainduration.

Kubernetes version: 1.28.6
Istio version: 1.18.7

**Comment by ramaraochavali** [comment, marked as answer]
Correct

**Reply by prodigy413** [reply]
Thank you!!!!!

**Comment by prodigy413** [comment]
> ###
> [ ](/ramaraochavali)



> Correct
